bonus_guides/G_static_assets.md

```diff
@@ -1 +1,289 @@
-### FPO
+Instead of implementing its own asset pipeline Phoenix uses [Brunch](http://brunch.io), a fast and developer-friendly asset build tool. Phoenix comes with a default configuration for Brunch and it will work out of the box, but it is very easy to bend it to our needs, add support for various script and style languages, like CoffeeScript, JSX, or LESS.
```

Comment:
I think we could use a comma in here:

. . . own asset pipeline, Phoenix uses . . .